ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksTable.tsx

```diff
@@ -0,0 +1,171 @@
+                                        <Td dataLabel="Check">
+                                            <TableText wrapModifier="wrap">
+                                                <Link
+                                                    to={generatePath(coverageCheckDetailsPath, {
+                                                        checkName,
+                                                        profileName,
+                                                    })}
+                                                >
+                                                    {checkName}
+                                                </Link>
+                                            </TableText>
+                                            <TableText wrapModifier="truncate">
+                                                <Text
+                                                    component={TextVariants.small}
+                                                    className="pf-v5-u-color-200"
+                                                >
+                                                    {rationale}
+                                                </Text>
+                                            </TableText>
+                                        </Td>
+                                        <Td dataLabel="Controls" modifier="fitContent">
+                                            placeholder
+                                        </Td>
+                                        <Td dataLabel="Fail status" modifier="fitContent">
+                                            <StatusCountIcon
+                                                text="cluster"
+                                                status="fail"
+                                                count={failCount}
+                                            />
+                                        </Td>
+                                        <Td dataLabel="Pass status" modifier="fitContent">
+                                            <StatusCountIcon
+                                                text="cluster"
+                                                status="pass"
+                                                count={passCount}
+                                            />
+                                        </Td>
+                                        <Td dataLabel="Other status" modifier="fitContent">
+                                            <StatusCountIcon
+                                                text="cluster"
+                                                status="other"
+                                                count={otherCount}
+                                            />
+                                        </Td>
+                                        <Td dataLabel="Compliance">
+                                            <Progress
```

Comment:
We can consider creating a component to encapsulate the rendering logic for the `Progress` and `Tooltip` components. Maybe we can also use the same string for the Progress component's `id` and the element id used for the `triggerRef` in `Tooltip` instead of constructing them separately. If one changes, we might miss changing the other.

Reply:
I was on the fence as to whether or not this deserved to be its own component but I definitely agree that `id` and `triggerRef` strings should be defined outside. I'll update the string for now and we can circle back on creating a new component.